_tools/redirects/redirects.csv

```diff
@@ -130,7 +130,7 @@ source,destination
 /getting_started/workflow/best_practices/scene_organization.html,/tutorials/best_practices/scene_organization.html
 /getting_started/workflow/best_practices/scenes_versus_scripts.html,/tutorials/best_practices/scenes_versus_scripts.html
 /getting_started/workflow/best_practices/what_are_godot_classes.html,/tutorials/best_practices/what_are_godot_classes.html
-/getting_started/workflow/export/android_custom_build.html,/tutorials/export/android_custom_build.html
+/getting_started/workflow/export/android_custom_build.html,/tutorials/export/android_gradle_build.html
```

Comment:
This means redirects would break for 4.0 and 4.1. I *think* adding a new second redirect with this instead should work fine, hopefully. Not 100% sure, it's been some time since messing with the redirects.

Reply:
> Alternatively (and potentially simpler) we could rename the page in 4.0 and 4.1 too and then have this redirect everywhere in 4.x, if it applies to all versions?

This change would apply to all 4.x versions so I can split this specific change in its own PR and add a cherry-pick for 4.0 and 4.1 as well.

Reply:
@mhilbrunner I've backported the changes in https://github.com/godotengine/godot-docs/pull/8217